mods/action.js

```diff
@@ -181,6 +181,13 @@ function buildEditResponse(res) {
 }
 
 ActionService.prototype._doRequest = function(restbase, req, defBody, cont) {
+    if (/\.test\.local$/.test(req.params.domain)) {
```

Comment:
We have to stop introducing these test hacks... Can't we have two action modules in the test config - one for en.wp and the other to be a *normal* one, and then use one or the other during the tests?